lib/src/features/line_renderer.dart

```diff
@@ -61,7 +69,48 @@ class LineRenderer extends FeatureRenderer {
         continue;
       }
 
-      context.canvas.drawPath(line, effectivePaint);
+      // do we need a dashed line?
```

Comment:
remove comments that are self-explanatory, prefer to write readable code instead